service/handlers.go

```diff
@@ -492,6 +492,39 @@ func newCelloToken(provider, key, secret string) *token {
 	}
 }
 
+// Convenience method that checks if a project exists using both the credential provider and database
```

Comment:
Per godoc, comments should begin with the name of the element it describes ie. `projectExists`.